std_msgs/Header header
geometry_msgs/Vector3 accelerometer_bias
geometry_msgs/Vector3 gyroscope_bias
